colorful/experimental-theme.json

```diff
@@ -0,0 +1,88 @@
+{
+    "global": {
+        "presets": {
+            "color": [
+                {
+                    "slug": "valencia",
+                    "value": "#DB403B"
+                },
+                {
+                    "slug": "mustard",
+                    "value": "#FFDE69"
+                },
+                {
+                    "slug": "navy",
+                    "value": "#022384"
+                }
+            ],
+            "font-size": [
+                {
+                    "slug": "small",
+                    "value": "12px"
+                },
+                {
+                    "slug": "medium",
+                    "value": "16px"
+                },
+                {
+                    "slug": "large",
+                    "value": "50px"
+                }
+            ],
+            "line-height": [
+                {
+                    "slug": "small",
+                    "value": "1.3"
+                },
+                {
+                    "slug": "medium",
+                    "value": "1.3"
+                },
+                {
+                    "slug": "large",
+                    "value": "1.4"
+                }
+            ],
+            "font-family": [
+                {
+                    "slug": "primary",
+                    "value": "'Jost', sans-serif"
+                },
+                {
+                    "slug": "secondary",
+                    "value": "'Lora', serif"
+                }
+            ],
+            "gradient": [
+            ]
+        },
+        "features": {
+            "typography": {
+                "dropCap": true
+            }
+        }
+    },
+    "core/paragraph": {
+        "styles": {
+            "color": "var(--wp--preset--color--valencia)",
+            "font-size": "var(--wp--preset--font-size--medium)"
+        }
+    },
+    "core/group": {
+        "styles": {
+            "color": "var(--wp--preset--color--valencia)"
+        }
+    },
+    "core/heading/h1": {
```

Comment:
I'm not sure if I see how this does anything. Removing it does not change anything relating to the output or color of an H1 tag. @jffng Would you be able to help me understand what this is doing? 🙏

<img width="227" alt="Screen Shot 2020-06-19 at 1 48 08 PM" src="https://user-images.githubusercontent.com/7538525/85165256-8a349700-b233-11ea-85f7-06ff692c8c61.png">

Reply:
Yeah, I haven't been able to get this part to work either (both here and in another theme I tried this with).

Reply:
I think the API changed since I first opened this PR. Take a look now?